src/getCoinCombination.js

```diff
@@ -10,6 +10,14 @@ function getCoinCombination(cents) {
   const values = [1, 5, 10, 25];
   const coins = [0, 0, 0, 0];
 
+  if (isNaN(cents)) {
+    return coins;
+  }
```

Comment:
The check for `isNaN(cents)` is redundant because the subsequent check for `typeof cents !== 'number'` will also handle non-numeric inputs. Consider removing the `isNaN` check to simplify the code.